realm/realm-jni/src/Makefile

```diff
@@ -3,7 +3,7 @@ lib_LIBRARIES = librealm-jni.a
 JNI_SOURCES := $(wildcard *.cpp)
 librealm_jni_a_SOURCES = $(JNI_SOURCES)
 
-# Used by ../../build.sh
+# Used by ../build.gradle
 get-inst-libraries:
```

Comment:
Is this target really used? It seems to be unused.
What do you think? @beeender